packages/client/src/components/App.tsx

```diff
@@ -32,9 +32,13 @@ export default class App extends React.Component<AppProps> {
               <div className="content-container-inner-wrapper">
                 <Route exact={true} path="/" component={Landing} />
                 <Route path="/docs/:path(.*)" component={Docs} />
-                <Route path="/config/edit/local" component={ConfigEditor} />
-                <Route path="/dev-tools/canary-executor" component={CanaryExecutor} />
-                <Route path="/dev-tools/results" component={CanaryExecutorResults} />
+                <Route path="/config/edit" component={ConfigEditor} />
+                <Route path="/dev-tools/canary-executor/setup" component={CanaryExecutor} />
+                <Route
+                  exact
+                  path="/dev-tools/canary-executor/results\/([0-9A-Z]{26})*"
```

Comment:
There is no contract stating that the IDs are 26 chars long, I think `\w+` would be better.

Reply:
Also I think you want to do it something like this

`/docs/:path(.*)` where the var is named

Reply:
implemented